modules/codelite/codelite_project.lua

```diff
@@ -188,6 +188,42 @@
 		}
 	end
 
+	local function getpch(cfg)
+		-- If there is no header, or if PCH has been disabled, I can early out
```

Comment:
That function is similar to the one from gmake/gmake2.
Not sure where common code should be factorized.

Reply:
If the code is exactly the same, you could `require()` in the gmake module here and then just call that implementation directly? Or it could be moved to a helper method in `gcc.lua` perhaps?

Reply:
Moved in gcc.